web/impact/impact/tests/test_snake_and_camel_casing.py

```diff
@@ -0,0 +1,16 @@
+from impact.tests.api_test_case import APITestCase
+from impact.models.utils import (
+    camel_case_to_snake,
+    snake_to_camel_case,
+)
+
+camel_case = 'ProgramStartupStatus'
+snake_case = 'program_startup_status'
+
+
+class TestSnakeAndCamelCasing(APITestCase):
+    def test_camel_case_to_snake_case(self):
+        assert camel_case_to_snake(camel_case) == snake_case
+
+    def test_snake_case_to_camel_case(self):
+        assert snake_to_camel_case(snake_case) == camel_case
```

Comment:
Was very glad to see those unit tests here!
I'd love to see also a test on an empty string, and lower-case word, and see that they're handled as expected.

Reply:
Can you clarify what you are looking for with "lower-case word"?  The snake case already checks lower case...